.github/ISSUE_TEMPLATE/intent-to-ship--i2s-.md

```diff
@@ -51,8 +51,66 @@ Provide instructions for how to demo your feature such as a link to a demo page.
 Add any other information that may be relevant in determining if your feature can ship.
 -->
 
+## Build checklist
+
+This section describes considerations the implementor should keep in mind while building this technical solution.
+
+- [ ] Tests written
+<!--
+Have unit tests been written? Integration tests? Validator tests?
+Details on unit and integration tests:https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#testing-your-changes
+Details on validator tests: https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md
+-->
+- [ ] Documentation present
+<!--
+Has reference documentation been added? Have examples been added? Has an example been made available on amp.dev?
+-->
+- [ ] Cross platform testing
+<!--
+Have you tested all browsers and versions that AMP supports? https://amp.dev/support/faq/supported-browsers/
+Have you tested across mobile and desktop devices? 
+-->
+
+### Considerations checklist
+
+This section describes different cross cutting considerations that the implementor should keep in mind while building this technical solution.
+
+<!--
+Retain all the considerations below that are applicable to your issue. If a consideration doesn't apply to you please remove that section.
+-->
+
+- #### Accessibility Considerations
+<!--
+Enter a short snippet for the accessibility considerations of the design. If none apply, enter n/a including a short sentence if not obvious. If you need assistance, reach out to the UI and Accessibility WG for assistance. https://github.com/ampproject/wg-ui-and-a11y
```

Comment:
I think that the second sentence (which I realize was copied from me) could use some word-tweaking.

Reply:
```suggestion
Enter a short snippet for the accessibility considerations of this design. If none apply, enter N/A along with a short explanation. If you need assistance, reach out to the UI and Accessibility WG: https://github.com/ampproject/wg-ui-and-a11y
```